src/components/Pages/MyAssets/MintNft/MintNft.tsx

```diff
@@ -2,24 +2,20 @@ import { BN } from '@polkadot/util';
 import { ChangeEvent, FormEvent, memo, useCallback, useEffect, useRef, useState } from 'react';
 import { FormControl, Stack } from 'react-bootstrap';
 import { Link, useParams } from 'react-router-dom';
+import { useZxing } from "react-zxing";
 
 import { saveImageToIpfs } from '@api/pinata.ts';
-
```

Comment:
Please leave these new lines,
yarn format does the job automatically for this part
but the ones in HTML below were done manually by me for clarity, I like to separate blocks visually, so leave them be